bindings/bindings.py

```diff
@@ -0,0 +1,558 @@
+import abc
```

Comment:
that's not used; this file itself is not under mypy so there's a few "linter issues" here and there.